src/java/org/streamingpool/core/service/impl/LocalPool.java

```diff
@@ -59,9 +65,18 @@ public LocalPool() {
     }
 
     public LocalPool(List<StreamFactory> factories) {
+        this(factories, defaultScheduler());
+    }
+
+    private static Scheduler defaultScheduler() {
```

Comment:
Also this is confusing ... Why is the default scheduler created here? I thought it comes from the configuration?

... I would go for only one default (configuration or default constructor)